crab/submit.py

```diff
@@ -73,9 +75,9 @@ def verifySample(sample):
 config.section_("JobType")
 config.JobType.pluginName = 'Analysis'
 config.JobType.psetName = '../NANOProducer/test/produceNANO.py'
-config.JobType.scriptArgs = [isData, 'year={}'.format(args.year)]
+config.JobType.outputFiles = ['nanox.root']
 #config.JobType.inputFiles = ['']
-config.JobType.maxMemoryMB = 2500
+config.JobType.maxMemoryMB = 2000
```

Comment:
note that the default limit is 2500 MB; so is there a need to lower this?

Reply:
Not really, crab complained about not using more than 70% of memory.